index.js

```diff
@@ -17,6 +17,38 @@ function Changelog( options ) {
 	this.options = options;
 	this.repo = new Repo( this.options.repo );
 
+	var ticketTypes;
+	if ( options.ticketTypes ) {
+		ticketTypes = options.ticketTypes;
+	} else {
+		ticketTypes = [ "github" ];
```

Comment:
The default should be `[ "github", "jira" ]`.

Make an object of default options and merge it into the provided options.